### PR TITLE
[Bulk| Validations: Fix missing alt text issues 9

### PR DESCRIPTION
**Pull request by v-lkasturi**
- Added alternate texts to all the screenshots.
- Convert absolute link to relative link.

### CODE REVIEW

File: iis/configuration/system.webServer/fastCgi/application/environmentVariables/index.md [hunk 1]
```diff
@@ -174,16 +174,16 @@ To use the FastCGI environment, you must first install the CGI role service. The
         > [!NOTE]
         > For PHP, this number must be lesser or equal to the maximum number of requests for PHP environment, which is configured by using the PHP\_FCGI\_MAX\_REQUESTS environment variable.
   
-   [![](index/_static/image24.png)](index/_static/image23.png)
+   [![Screenshot shows the Add Fast C G I Application window with the Full Path and Arguments fields.](index/_static/image24.png)](index/_static/image23.png)
 6. Click the ellipsis (**...**) next to **EnvironmentVariables** to open the environment variables collection editor.
 7. In the **EnvironmentVariables Collection Editor** dialog box, click **Add**.  
-   [![](index/_static/image26.png)](index/_static/image25.png)
+   [![Screenshot shows the EnvironmentVariables Collection Editor dialog box to add Members.](index/_static/image26.png)](index/_static/image25.png)
```
Comment (RedZephyr13) [inline iis/configuration/system.webServer/fastCgi/application/environmentVariables/index.md:180]: I think separating the individual words for "EnvironmentVariables" would make sense here.

File: iis/configuration/system.webServer/fastCgi/application/environmentVariables/index.md [hunk 2]
```diff
@@ -174,16 +174,16 @@ To use the FastCGI environment, you must first install the CGI role service. The
...
 8. Enter the following information, and then click **OK**.
 
    - Enter "PHP\_FCGI\_MAX\_REQUESTS" for the **Name**.
    - Enter "10000" for the **Value**.
   
-   [![](index/_static/image28.png)](index/_static/image27.png)
+   [![Screenshot shows the EnvironmentVariables Collection Editor dialog box highlighting the Name.](index/_static/image28.png)](index/_static/image27.png)
```
Comment (RedZephyr13) [inline iis/configuration/system.webServer/fastCgi/application/environmentVariables/index.md:186]: I think separating the individual words for "EnvironmentVariables" would make sense here.

**Comment by v-lkasturi** [comment]
@Rick-Anderson @mcleblanc The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

CC: @MonicaRush @snoviking

**Review by Rick-Anderson — APPROVED** [review]

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
Can you fix the warnings in the build report.

**Comment by v-lkasturi** [comment]
@Rick-Anderson @mcleblanc The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

CC: @MonicaRush @snoviking

**Comment by Rick-Anderson** [comment]
@v-lkasturi @RedZephyr13 Can you fix the warnings in the build report.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**